package.json

```diff
@@ -99,7 +99,9 @@
     "lint-staged": "lint-staged",
     "build": "dotnet build build/build-dotnet.sln && gulp default",
     "build-dist": "dotnet build build/build-dotnet.sln && gulp default --uglify",
-    "build-themes": "gulp style-compiler-themes"
+    "build-themes": "gulp style-compiler-themes",
+    "dev": "gulp dev",
+    "test": ".\\testing\\launch"
```

Comment:
🤷‍♂️ 
`test` isn't a task.
It launches UI and requires interaction.
I suggest that we gather more opinions.

Reply:
Not cross-platform due to Win-style slashes.
`node testing/launch`?

Reply:
Maybe rename this task to 'serve-test' or 'serve-qunit?

Reply:
Yes, I know, we can name it `test-launch` or something like that, but so many chars...

Reply:
>so many chars...
How about `test-env`? :)

Reply:
`test-env` sounds good